### PR TITLE
Mentors page misc fixes

### PR DESCRIPTION
**Pull request by tandebnath**

### CODE REVIEW

**Review by ashayp22 — APPROVED** [review]

**Comment by lasyaneti** [comment]
DONT MERGE YET

File: components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -55,6 +55,10 @@ const Navbar = () => {
                         {
```
Comment (ashayp22) [inline components/Navbar/Navbar.tsx:59]: Forgot to show Mentors above when the user is authenticated and registered.

**Review by ashayp22 — CHANGES_REQUESTED** [review]
One small fix needed.

File: components/Navbar/Navbar.tsx (hunk 1 above)
Comment (ashayp22) [reply components/Navbar/Navbar.tsx:59]: Instead, remove adding the Mentors page here and move it to `DEFAULT_NAVBAR_ITEMS` at the top of the screen.

**Review by nrking0 — APPROVED** [review]

**Pull request merged**